src/models/message.js

```diff
@@ -1,16 +1,15 @@
-const message = (sequelize, DataTypes) => {
-  const Message = sequelize.define('message', {
-    text: {
-      type: DataTypes.STRING,
-      validate: { notEmpty: true },
-    },
-  });
+import mongoose from 'mongoose'
 
-  Message.associate = models => {
-    Message.belongsTo(models.User);
-  };
-
-  return Message;
-};
-
-export default message;
+const Message = mongoose.model('Message', new mongoose.Schema({
+  text: {
+    type: String,
+    required: true
+  },
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'User'
+  }
+}, {
+  timestamps: true
```

Comment:
Does this mean createdAt and editedAt are automatically included?

Reply:
yes